llvm/include/llvm/IR/CFG.h

```diff
@@ -391,12 +423,24 @@ template <> struct GraphTraits<Inverse<Function*>> :
   static NodeRef getEntryNode(Inverse<Function *> G) {
     return &G.Graph->getEntryBlock();
   }
+
+  static unsigned getMaxNumber(Function *F) { return F->getMaxBlockNumber(); }
+  static unsigned getNumberEpoch(Function *F) {
```

Comment:
Missing const here and in other declarations
```suggestion
  static unsigned getNumberEpoch(const Function *F) {
```